pkg/storaged/content-views.jsx

```diff
@@ -497,7 +497,8 @@ function append_partitions(client, rows, level, block) {
                     {utils.format_size_and_text(size, _("Free space"))}
                 </span>
             },
-            { title : btn }
+            { },
+            { title : btn, props: { className: "content-action" } }
```

Comment:
content action class can be replaced with pf-c-table__action PF4 class. the effect is slightly different, but I am satisfied with their solution as well.

Reply:
Looks like this with pf-c-table__action:

![image](https://user-images.githubusercontent.com/3228183/124112881-742afa80-da73-11eb-9f8d-146f6115f65a.png)

I would keep the content-action class:

![image](https://user-images.githubusercontent.com/3228183/124105844-4ee6be00-da6c-11eb-9c00-31b371724ce3.png)

Reply:
It's still different than what PF offers here with buttons and kebab but sure, it's not a new class, so I am fine with it.
https://www.patternfly.org/v4/components/table/#composable-actions